test/test_transformers.py

```diff
@@ -1637,6 +1639,8 @@ def test_invalid_fused_inputs_attn_mask_present(self, device, kernel: SDPBackend
     @onlyCUDA
     @unittest.skipIf(not PLATFORM_SUPPORTS_FLASH_ATTENTION, "Does not support fused SDPA or pre-SM80 hardware")
     def test_unaligned_tensors(self, device):
+        if TEST_WITH_ROCM:  # ROCm GPU kernels does not have strict alignment requirements
+            return
```

Comment:
Can you please explain what this line supposed to do? If GPUs does not have strict alignment requirements, why skip the test? Also, if you do want to skip, use standard decorator or use `raise unittest.SkipTest("Reason")`

Reply:
If we keep running the testing code, the test will fail if not skipped, because the test's logic is asking for a RuntimeError which we will not raise.
The effect of this line is to mark the test passed rather than skipped.

@jithunnair-amd  what is the canonical solution for this situation? Marking as skipped will create troubles when examining UT parities.

Reply:
@malfet updated with a new decorator `xfailIfRocm` to indicate the UT is expected to fail on ROCM.

Reply:
Yes, I think xfailing on ROCm is the right approach